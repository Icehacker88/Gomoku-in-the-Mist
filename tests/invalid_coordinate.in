place A0
place A20
place A02
place A002
place A0020
place a1
place A1A1
place A111
place nevergonnagiveyouup
place nervergonnasaygoodbye
place Z1
place B123
term
