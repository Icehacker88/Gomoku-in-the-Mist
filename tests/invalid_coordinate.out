Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
Invalid coordinate
